frontend/views/pages/Page.vue

```diff
@@ -27,7 +33,8 @@ export default {
   },
   props: {
     pageTestName: String,
-    pageTestHeaderName: String
+    pageTestHeaderName: String,
```

Comment:
Instead of having `pageTestName` wouldn't it be simpler to spread the props?
And the `pageTestHeaderName` could be automatic:
```
data-test="`${['data-test']}-title`"
```

ex:
<Page data-test="payGroup" />

// data-test -> `payGroup`
// title data-test -> `payGroup-title`